### DISCUSSION TITLE
Now Playing not shown at an Icecast FLAC stream

### DISCUSSION THREAD

**Discussion by Piratenkanon** [AzuraCast/AzuraCast#6386] [Q&A]
Hello to all,
i created a FLAC-Stream, realized on an Icecast server.
The Now Playing ist not shown "Station Offline" is shown, but playing!!
Next Song is working.
what's goin wrong??
![now_playing](https://github.com/AzuraCast/AzuraCast/assets/62739604/22b614e1-6dae-4f66-b04e-9e03d27ab646)

**Comment by Citizen3rased** [comment]
I'm having this same issue specifically with FLAC streaming, but with AutoDJ as well.
